Task 1/Even Odd/Question.md

```diff
@@ -0,0 +1 @@
+Using JavaScript when a number is entered into an input field, it should display above the input either as `This number you typed is an even number` or `This number you typed is an odd number`.
```

Comment:
Elaborate giving 1-2 test cases and headings/subheadings, constraints on the input number etc.